public/css/app.css

```diff
@@ -651,14 +651,19 @@ form span {
   justify-content: center;
   align-items: center;
   background-color: rgba(0, 0, 0, 0.38);
+  transition: all 0.25s ease-out;
```

Comment:
microscopic nit: but my eyes are noting that almost all other transitions seem to be 0.2s and this is an outlier at 0.25s.
That's probably fine and all, just noting the difference.

```sh
$ git grep -n "transition: "

public/css/app.css:87:  transition: all cubic-bezier(0.07, 0.95, 0, 1);
public/css/app.css:91:  transition: all cubic-bezier(0.07, 0.95, 0, 1);
public/css/app.css:112:  transition: box-shadow 0.2s, border-color 0.2s ease-in-out;
public/css/app.css:136:  transition: all 0.2s ease-out;
public/css/app.css:449:  transition: background-color 0.2s ease-out, color 0.2s ease-out;
public/css/app.css:519:  transition: all 0.2s ease-out;
public/css/app.css:575:  transition: all 0.2s ease-out;
public/css/app.css:580:  transition: all 0.2s ease-out;
public/css/app.css:654:  transition: all 0.25s ease-out;
public/css/app.css:666:  transition: all 0.2s ease-out;
public/css/app.css:685:  transition: all 0.2s ease;
public/css/app.css:937:  transition: all 0.2s ease-out;
```